pkg/pgmodel/ingestor/trace/tag_batch_test.go

```diff
@@ -121,23 +121,23 @@ func TestTagBatch(t *testing.T) {
 			getTagMapJSONCheck: func(t *testing.T, batch tagBatch) {
 				tagMap, err := batch.GetTagMapJSON(map[string]interface{}{"test": ""}, SpanTagType)
 				require.Nil(t, err)
-				require.Equal(t, `{"7":8}`, string(tagMap))
+				require.Equal(t, `{"7":8}`, string(tagMap.Bytes))
 
 				tagMap, err = batch.GetTagMapJSON(map[string]interface{}{"nonexistant": ""}, SpanTagType)
 				require.EqualError(t, err, "error getting tag from batch {nonexistant \"\" 1}: error getting item from batch")
-				require.Equal(t, []byte(nil), tagMap)
+				require.Equal(t, []byte(nil), tagMap.Bytes)
```

Comment:
If we return nil tagMap pointer (instead of default struct) this check would be simpler: `require.Nil(t, tagMap)`